server/package.json

```diff
@@ -15,7 +15,7 @@
     "lint": "tslint -c ../tslint.json  src/**/*.ts -t stylish",
     "test": "npm run build && RECORD=false npm run test:ava -- --timeout=10sec && npm run check-coverage",
     "test:record": "npm run build && RECORD=true npm run test:ava -- --timeout=1min",
-    "test:ava": "nyc ava lib/test/**/*-test.js --verbose"
+    "test:ava": "nyc --reporter=text --reporter=html ava lib/test/**/*-test.js --verbose"
```

Comment:
What does specifying both do?

Reply:
applying multiple reporters means NYC will report in both styles. In this case it'll continue to print the summary but it will also create a html report (which I personally find helpful to see what code isn't being covered vs any of the text versions)

Reply:
Cool, sg!